<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1303</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="menu">
   <widget class="QTabWidget" name="tabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1301</width>
      <height>691</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_preprocessing">
     <attribute name="title">
      <string>Preprocessing</string>
     </attribute>
     <widget class="QTabWidget" name="subtabs">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1301</width>
        <height>671</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="AdjustBrightness">
       <attribute name="title">
        <string>Adjust Brightness --&gt;</string>
       </attribute>
       <widget class="QGroupBox" name="pre_input_img">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>611</width>
          <height>491</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>INPUT</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="pre_ImgWidget" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>611</width>
           <height>471</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="pre_adjbr_toggle">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>TURN ON</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>1037</x>
          <y>20</y>
          <width>161</width>
          <height>191</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="pre_brightness">
             <property name="text">
              <string>Brightness</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="pre_brightness_slider">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="pre_brightness_box">
             <property name="suffix">
              <string> %</string>
             </property>
             <property name="prefix">
              <string/>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="pre_contrast">
             <property name="text">
              <string>Contrast</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="pre_contrast_slider">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="pre_contrast_box">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="suffix">
              <string> %</string>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="stepType">
              <enum>QAbstractSpinBox::DefaultStepType</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pre_adjbr_switch">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>40</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>IN/OUT</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Greyscale">
       <attribute name="title">
        <string>Greyscale --&gt;</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="Filtering">
       <attribute name="title">
        <string>Filtering --&gt;</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_panel_detection">
     <attribute name="title">
      <string>Panel Detection</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_button_detection">
     <attribute name="title">
      <string>Button Detection</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_roi_extraction">
     <attribute name="title">
      <string>ROI Extraction</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_button_recognition">
     <attribute name="title">
      <string>Button Recognition</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_postprocessing">
     <attribute name="title">
      <string>Postprocessing</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_visual_tracking">
     <attribute name="title">
      <string>Visual Tracking</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_settings">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>500</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Apply|QDialogButtonBox::Discard|QDialogButtonBox::RestoreDefaults</set>
      </property>
      <property name="centerButtons">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>320</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>290</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>290</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>220</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>410</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label1">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>280</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Mylabel</string>
      </property>
     </widget>
     <widget class="QOpenGLWidget" name="openGLWidget">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>300</width>
        <height>200</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1303</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tabs</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pre_brightness_box</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pre_brightness_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1043</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>1043</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pre_brightness_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pre_brightness_box</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1043</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>1043</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pre_contrast_box</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pre_contrast_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1121</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>1113</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pre_contrast_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pre_contrast_box</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1168</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>1168</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
